code-challenges.js

```diff
@@ -7,43 +7,224 @@
+
+//tough one here. 
+//i believe i want todo an iteration using For loop.
+//the count starts at O
+//how many iterations i want the loop to do??? 6?
+//taking each number and dividing bt 2 to determine if its even or not
+//if it doesnt equal to 0 then its odd
+//else if it does then its even
+
+
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// for(let i = 0; i<myArray.length; i++){
+//  if(myArray[i] % 2 !==0){
+//    console.log(`"odd"`)
+//  } else{
+//      console.log(`"even"`)
+//  }
+// }
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js
+// "odd"
+// "even"
+// "odd"
+// "even"
+// "odd"
+// "even"
+
+//ALL THE PREVIOUS ATTEMPTS, I FORGOT TO WRITE DOWN MY PROCESSES.
+// 13 is an odd number
+// 5 is an odd number
+// 27 is an odd number
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13 odd
+// 5 odd
+// 27 odd
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13 odd
+// 34 even
+// 5 odd
+// 10 even
+// 27 odd
+// 42 even
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13 "odd"
+// 34 "even"
+// 5 "odd"
+// 10 "even"
+// 27 "odd"
+// 42 "even"
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// 13,34,5,10,27,42 "odd"
+// 13,34,5,10,27,42 "even"
+// learnacademy@LEARNs-Air-4 week-1-assessment-GeeeShep % node code-challenges.js node code-challenges.js
+// 13 "odd"
+// 34 "even"
+// 5 "odd"
+// 10 "even"
+// 27 "odd"
+// 42 "even"
+//-----------------------------
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
-// Set one:
-const number1 = 58
-const number2 = 100
+// so i know i need to get an output of 42 and 3 by subtracting the two constants in each set
+//i first tried console.log(number2 - number1) which gave me 42.
+//now i can put them together in a string to get the output on the same line
 
-// Set two:
-const number3 = 27
-const number4 = 24
+// // Set one:
+ const number1 = 58
+ const number2 = 100
+
+// // Set two:
+ const number3 = 27
+ const number4 = 24
+
+ //console.log(number2 - number1)
+
+//console.log(number2 - number1, (number3 - number4))
```

Comment:
In this case, you would want to create a function so we can have some dynamic code.  Meaning we can have any set of 2 numbers be passed in and the function will handle the output properly.  You likely will need to use a conditional to check which number is larger and subtract the smaller number from the larger number.